pages/about.html

```diff
@@ -0,0 +1,226 @@
+        <li class="nav-item">
+          <a class="nav-link" href="#">
+            <img src="./images/linke.png" alt="linkedin" />
+          </a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#">
+            <img src="./images/twitter.png" alt="twitter" />
+          </a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#">English</a>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#">My Page</a>
+        </li>
+        <li class="nav-item navLogout">
+          <a class="nav-link" href="#">Logout</a>
+        </li>
+      </ul>
+    </nav>
+
+    <!-- // navbar Section // -->
+
+    <nav class="navbar navbar-expand-sm bg-light navbar-light">
+      <div class="container">
+        <a class="navbar-brand" href="#">
+          <img src="./images/Enha (1).png" alt="logo" id="logoMain" />
+        </a>
+        <button
+          class="navbar-toggler main_btn"
+          type="button"
+          data-bs-toggle="collapse"
+          data-bs-target="#collapsibleNavbar"
+        >
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div
+          class="collapse navbar-collapse justify-content-end"
+          id="collapsibleNavbar"
+        >
+          <ul id="menyOptions" class="navbar-nav nav2">
+            <li class="nav-item">
+              <a class="nav-link" href="../index.html">Home</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="./about.html">About</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="#partners">Partners</a>
+            </li>
+          </ul>
+          <ul id="popup" class="navbar-nav nav2">
+            <div id="cross-icon">
+              <!-- <img
+                onclick="off()"
+                class="style-li"
+                src="/close-Icon.png"
+                alt="close-Icon"
+              /> -->
+              <i
+                onclick="off()"
+                class="material-icons"
+                style="font-size: 50px; color: rgb(226, 215, 215);"
+              >
+                close
+              </i>
+            </div>
+            <li onclick="off()" class="nav-item">
+              <a class="nav-link li-nav" href="../index.html">Home</a>
+            </li>
+            <li onclick="off()" class="nav-item">
+              <a class="nav-link li-nav" href="./about.html">About</a>
+            </li>
+            <li onclick="off()" class="nav-item">
+              <a class="nav-link li-nav" href="#partners">Partners</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+
+    <!-- // Main Section // -->
+
+    <div id="main1" class="container-fluid">
+      <div class="d-flex pt-5 px-1 justify-content-center">
+        <h1 style="color: red; font-weight: bold; font-size: xxx-large;">
+          Enhance Your Skills
+        </h1>
```

Comment:
- [ ] This is a subtitle and as such should use h2, h3 h4 e.t.c for semantic purposes